packages/swirl-components/src/components/swirl-shell-layout/swirl-shell-layout.css

```diff
@@ -134,22 +135,19 @@
   display: flex;
   overflow: hidden;
   align-items: center;
+  height: 4.5rem;
```

Comment:
**issue:** Spacing to the navigation is off.
<img width="65" alt="Bildschirm­foto 2023-03-06 um 09 06 44" src="https://user-images.githubusercontent.com/1019414/223052821-e26ec03a-02cd-4677-bf4b-3b2c31761499.png">